### DISCUSSION TITLE
How to change the position of the gradient when reversing the y axis?

### DISCUSSION THREAD

**Discussion by danifox-s** [apexcharts/apexcharts.js#2929] [Q&A]
I have reversed Y axis by doing this:
`yaxis: {
    reversed: true,
}`
And I'm getting this:
![image](https://user-images.githubusercontent.com/85786951/151232272-ae30e401-e392-4458-8ece-c15fbb055e12.png)
Is it possible to change the position of the gradient so that it is at the bottom?

**Comment by andrewl94** [comment]
Does not seems like is currently possible.
This closed issue #1337 discussed a possible "reverseFill" option, but I dont see anything on documentation.
